kratky_rg_izero.py

```diff
@@ -169,21 +169,20 @@ def rg_i0(argv):
     >>> python3 RG_and_IO.py my/data/dir -o save/to/file -c red blue
     """
     parser = ArgumentParser(description="Create scatterplot of data.")
-    parser.add_argument('mydir', metavar='DIR',
+    parser.add_argument('mydir', nargs='?', metavar='DIR',
                         help='Path to directory containing data',
                         default=os.getcwd())
-    parser.add_argument('-outfile', '-o',
+    parser.add_argument('--outfile', '-o',
                         help='Filepath including filename for plot, no ext.')
-    parser.add_argument('-colors', '-c', nargs='+',
+    parser.add_argument('--colors', '-c', nargs='+',
                         help="Colors for scatterplot",
                         default=['#e41a1c', '#377eb8', '#4daf4a', '#984ea3',
                                  '#984ea3', '#ffff33'])
```

Comment:
I have a suggestion here. nargs='+' can be a bit weird for optional arguments. It is most typically used when providing a list of filenames on the command line that all must be processed. It is less commonly used for optional args.

If I were writing this, I would actually do something like the following:
```python
import matplotlib.colors as mcolors

def colors(colorstring, sep=','):
    ''' Split a comma-separated list of strings and check that they are valid colors.
        @returns list of strings
    '''
    values = []
    for v in colorstring.split(sep):
        if mcolors.is_color_like(v):
            values.append(v)
        else:
            raise ArgumentError('Invalid value {}, value must be a valid color'.format(v))
    return values

...
    # in rg_io function
    parser.add_argument('--colors', '-c', nargs=1, help="Comma-separated colors for scatterplot", type=colors)
```

This is a pre-existing issue, so I'm fine moving to a followup PR focused just on that change.